.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
   - wget -qO - https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
   - sudo apt-get -qq update
   # Install dependencies
-  - sudo apt -y install clang-7 libssl-dev gdb libsgx-enclave-common libsgx-enclave-common-dev libprotobuf10 libsgx-dcap-ql libsgx-dcap-ql-dev az-dcap-client open-enclave libmbedtls-dev
+  - sudo apt -y install clang-7 libssl-dev gdb libsgx-enclave-common libsgx-enclave-common-dev libprotobuf10 libsgx-dcap-ql libsgx-dcap-ql-dev az-dcap-client open-enclave=0.8.2 libmbedtls-dev
```

Comment:
We support 0.9 as well

Reply:
Right. I just specified 0.8.2 in the CI build because 0.9 [begins deprecation](https://github.com/openenclave/openenclave/releases/tag/v0.9.0) of the `oehostapp` target. I suppose 0.9 is better though since it adds more features but doesn't completely drop support for `oehostapp`. I'll change the version to 0.9.